sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/WindowsUpdateClass.cs

```diff
@@ -10,7 +10,10 @@
 
 namespace Azure.ResourceManager.Automation.Models
 {
-    /// <summary> Update classification included in the software update configuration. A comma separated string with required values. </summary>
+    /// <summary>
+    /// Update classification included in the software update configuration. A comma separated string with required values
+    /// Serialized Name: WindowsUpdateClasses
+    /// </summary>
     public readonly partial struct WindowsUpdateClass : IEquatable<WindowsUpdateClass>
```

Comment:
Suggest renaming to `WindowsUpdateClassification`

Reply:
fixed